src/main/java/me/arminb/sara/controller/models/AnswerRequest.java

```diff
@@ -0,0 +1,69 @@
+package me.arminb.sara.controller.models;
+
+import me.arminb.sara.entities.Answer;
+import me.arminb.sara.entities.Comment;
+
+import java.util.List;
+
+
+public class AnswerRequest {
+
+    private String answer;
+
+    private int rate;
+
+    private List<Comment> comments;
+
+    private String user;
+
+    private String questionId;
```

Comment:
questionId will be passed through a path variable. I would remove it from this request class